### PR TITLE
simplify `next-binding`

### PR DESCRIPTION
**Pull request by ForsakenHarmony**

### CODE REVIEW

**Comment by ForsakenHarmony** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2899** <a href="https://app.graphite.dev/github/pr/vercel/turbo/2899" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/vercel/turbo/2899?utm_source=stack-comment).

File: crates/next-binding/src/lib.rs [hunk 1]
```diff
@@ -1,53 +1,34 @@
...
-    pub mod dev_server {
-        pub use next_dev::*;
-    }
-
-    #[cfg(feature = "__feature_node_file_trace")]
-    pub mod node_file_trace {
-        pub use node_file_trace::*;
-    }
+    #[cfg(feature = "__turbo_next_dev_server")]
+    pub use next_dev as next_dev_server;
```
Comment (kwonoj) [inline crates/next-binding/src/lib.rs:25]: I just realized this is probably my mistake to rename imports, the intention is transparently reexports without rename. Mind to change to export it as original name?

File: crates/next-binding/src/lib.rs [hunk 2]
```diff
@@ -1,53 +1,34 @@
...
+    pub use node_file_trace;
 }
 
 #[cfg(feature = "__features")]
 pub mod features {
     #[cfg(feature = "__feature_mdx_rs")]
-    pub mod mdx {
-        pub use mdxjs::*;
-    }
+    pub use mdxjs as mdx;
```
Comment (kwonoj) [inline crates/next-binding/src/lib.rs:33]: same as next_dev

**Review by kwonoj — APPROVED** [review]
👍

File: crates/next-binding/src/lib.rs (hunk 1 above)
Comment (ForsakenHarmony) [reply crates/next-binding/src/lib.rs]: But `swc::core` stays?

Comment (ForsakenHarmony) [reply crates/next-binding/src/lib.rs]: done

Comment (kwonoj) [reply crates/next-binding/src/lib.rs]: that I need to think bit, I felt bit redundant to make `swc::swc_core` 🤔

**Review by kwonoj — APPROVED** [review]

File: crates/next-binding/src/lib.rs (hunk 1 above)
Comment (ForsakenHarmony) [reply crates/next-binding/src/lib.rs]: true

**Pull request merged**